Tasks/Gradle/CodeAnalysis/SonarQube/endpoint.ts

```diff
@@ -15,6 +15,8 @@ export class SonarQubeEndpoint {
         var genericEndpointName: string = tl.getInput('sqConnectedServiceName');
         var hostUrl = tl.getEndpointUrl(genericEndpointName, false);
 
+        tl.debug(`[SQ] SonarQube endpoint: ${hostUrl}`);
```

Comment:
Not part of the fix - I have added a few more debug messages to better pinpoint problems in the future.